pkg/controller/operators/catalog/operator.go

```diff
@@ -392,6 +464,23 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 	op.lister.CoreV1().RegisterPodLister(metav1.NamespaceAll, csPodInformer.Lister())
 	sharedIndexInformers = append(sharedIndexInformers, csPodInformer.Informer())
 
+	if err := labelObjects(corev1.SchemeGroupVersion.WithResource("pods"), csPodInformer.Informer(), labeller.ObjectLabeler[*corev1.Pod, *corev1applyconfigurations.PodApplyConfiguration](
+		ctx, op.logger, func(object metav1.Object) bool {
+			if labels := object.GetLabels(); labels != nil {
+				if _, ok := labels[reconciler.CatalogSourceLabelKey]; ok {
```

Comment:
I take it the only pods we care about are those labeled as a catalog source?

Reply:
There's four sets of Pods we care about, but the rest all have dedicated informers already and are labeled. This particular one is just for catalog sources, yes.